rules/typescript.js

```diff
@@ -1,46 +1,6 @@
-module.exports = {
+export default {
   rules: {
     '@typescript-eslint/array-type': ['error', { default: 'array-simple' }],
-    '@typescript-eslint/ban-types': [
```

Comment:
this is now three separate rules that are enabled below in this file, see [here](https://typescript-eslint.io/blog/announcing-typescript-eslint-v8/#replacement-of-ban-types)